rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -251,7 +252,6 @@ SettingsDialog::SettingsDialog(wxWindow *parent, rpcs3::config_t* cfg)
 		chbox_gs_overlay->Enable(false);
 	}
 
-	cbox_gs_render->Append("Vulkan");
```

Comment:
Why was Vulkan inside the check for a Microsoft OS? (Wouldn't that make Vulkan not usable in Linux/Mac OSX?)